{"name": "Тест увеличения общего каунтера при оформлении заказа", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:511:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16", "trace": "self = <tests.test_lenta_orders.TestLenta object at 0x10418cb60>\n\n    @allure.title(\"Тест увеличения общего каунтера при оформлении заказа\")\n    @pytest.mark.usefixtures(\"driver\")\n    def test_counter_all_orders_with_my_order(self):\n        self.driver.get(main_url)\n        client = LentaPage(self.driver)\n>       result = client.check_counter_all_oders()\n\ntests/test_lenta_orders.py:41: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npages/lenta_orders_page.py:63: in check_counter_all_oders\n    self.wait(self.input_email)\npages/base_page.py:18: in wait\n    return WebDriverWait(self.driver, 10).until(EC.element_to_be_clickable(element))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"314fec51-2cdc-4ec3-9e2a-7e4e86bdc4bf\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x104249da0>, message = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       RemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nE       WebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nE       NoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:511:5\nE       dom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\nvenv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException"}, "steps": [{"name": "Проверка увеличения каунтера всех заказов", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:511:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\n", "trace": "  File \"/Users/ulabualskaa/Documents/GitHub/Diplom_3/venv/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/ulabualskaa/Documents/GitHub/Diplom_3/pages/lenta_orders_page.py\", line 63, in check_counter_all_oders\n    self.wait(self.input_email)\n  File \"/Users/ulabualskaa/Documents/GitHub/Diplom_3/pages/base_page.py\", line 18, in wait\n    return WebDriverWait(self.driver, 10).until(EC.element_to_be_clickable(element))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ulabualskaa/Documents/GitHub/Diplom_3/venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1727003906407, "stop": 1727003916860}], "parameters": [{"name": "driver", "value": "'firefox'"}], "start": 1727003906222, "stop": 1727003916861, "uuid": "d4e3589c-58a2-4006-9135-832561b95881", "historyId": "fdb723f66b2df95911b1354cd2933d5e", "testCaseId": "98af9a1c866c1b1478e96f80a5f5e6b4", "fullName": "tests.test_lenta_orders.TestLenta#test_counter_all_orders_with_my_order", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('driver')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_lenta_orders"}, {"name": "subSuite", "value": "TestLenta"}, {"name": "host", "value": "MacBook-Pro-Ula.local"}, {"name": "thread", "value": "8096-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_lenta_orders"}]}